powerline/listers/i3wm.py

```diff
@@ -0,0 +1,80 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+from powerline.lib.dict import updated
+
+
+conn = None
+XRANDR_OUTPUT_RE = re.compile('^(?P<name>[0-9A-Za-z-]+) connected (?P<width>\d+)x(?P<height>\d+)\+(?P<x>\d+)\+(?P<y>\d+)', re.MULTILINE)
```

Comment:
And this should be two blank lines away from get_connected_xrandr_outputs. I actually would like this close position, but AFAIR linter does not accept this.

(BTW, it is better to write `r'…'`: I do not like relying on `'\d'` being interpreted as `'\\d'`.)